File: ts/ping.ts
let rpcPing: nkruntime.RpcFunction = function (ctx: nkruntime.Context, logger: nkruntime.Logger, nk: nkruntime.Nakama, payload: string): string {
    logger.debug("Ping received with payload: %s", payload);
    return JSON.stringify("pong");
};

File: go/ping.go
package main

import (
	"context"
	"database/sql"

	"github.com/heroiclabs/nakama-common/runtime"
)

func rpcPing(ctx context.Context, logger runtime.Logger, db *sql.DB, nk runtime.NakamaModule, payload string) (string, error) {
	logger.Debug("Ping received with payload: %s", payload)
	return "pong", nil
}
